samples/testing/operations/src/OperationEquivalence.qs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/// # Sample
+/// Multi File Testing Project (Refer to README for Project Overview)
+///
+/// # Description
+/// This code builds upon the concepts explained in the README file,
+/// demonstrating how to organize Q# code into multiple files for testing.
+
+namespace OperationEquivalence {
+    open Microsoft.Quantum.Diagnostics;
+    open CustomOperation;
+    /// # Summary
+    /// Verifies the equivalence of quantum operations using `Fact` function
```

Comment:
Here, we need to mention that this check compares two operations up to a global phase. This is important, for example, if the user tries to hunt down a global phase issue. The other approach to testing doesn't have this issue